### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by nazarepiedady**
### Description

I updated the dependencies listed inside `package.json` and
I tested the website with the new versions of the dependencies,
and since everything is working fine I thought that would be good
to pull request the updates.

### CODE REVIEW

File: docs/.vitepress/config.ts [hunk 1]
```diff
@@ -91,6 +91,7 @@ export default defineConfig({
     zh: { label: '简体中文', link: 'https://cn.vitejs.dev' },
     ja: { label: '日本語', link: 'https://ja.vitejs.dev' },
     es: { label: 'Español', link: 'https://es.vitejs.dev' },
+    pt: { label: 'Português', link: 'https://pt.vitejs.dev' },
```
Comment (tony19) [inline docs/.vitepress/config.ts:94]: This site doesn't actually exist.
```suggestion
```

**Review by tony19 — CHANGES_REQUESTED** [review]

File: docs/.vitepress/config.ts (hunk 1 above)
Comment (nazarepiedady) [reply docs/.vitepress/config.ts:94]: @tony19 the source of the site is ready, it is waiting to be served
by the people from vite team. I already did the pull request to
inform the conclusion for the first version of the translated source.

So could you at least wait until the site is served? :+1:

Comment (tony19) [reply docs/.vitepress/config.ts:94]: Ok. When that happens, the change will automatically propagate to this repo (via a GitHub bot that merges changes from the main `vite` repo), so there's no need to add the locale explicitly here.

Comment (nazarepiedady) [reply docs/.vitepress/config.ts:94]: Good, I did not know about that :smile: .
@tony19 So can you only pick the others updates except this?

Comment (tony19) [reply docs/.vitepress/config.ts:94]: Yes, I prefer to keep the docs in sync with the upstream repo.

**Review by tony19 — APPROVED** [review]

**Pull request merged**